tests/aiocamedomotic/test_real.py

```
@@ -41,8 +41,11 @@
 
 from aiocamedomotic import CameDomoticAPI
 from aiocamedomotic.models import LightStatus
+from aiocamedomotic.models.opening import Opening, OpeningStatus, OpeningType
```

Ruff (0.8.2)

`aiocamedomotic.models.opening.Opening` imported but unused

Remove unused import

(F401)